docs/library/sys.rst

```diff
@@ -115,6 +115,13 @@ Constants
    If you need to check whether your program runs on MicroPython (vs other
    Python implementation), use `sys.implementation` instead.
 
+.. data:: settrace
+
+   Requires custom MicroPython compilation as it is ,ostly not present in
+   pre-built  firmware (due to its affecting performance). The relevant build
```

Comment:
Please remove the extra space between pre-built and firmware. 
Suggest: "(due to the negative effect on performance)"